GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -93,6 +93,7 @@ public void CancelBuildStatusFetchOperation()
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times")]
```

Comment:
:exclamation: This warning was not incorrect. The `MemoryStream` instance is disposed twice because the [`StreamWriter` constructor](https://msdn.microsoft.com/en-us/library/3aadshsx.aspx) takes ownership of the `Stream` it is created with.